source/Magritte-Pharo7-Model/MAFileDescription.extension.st

```diff
@@ -0,0 +1,14 @@
+Extension { #name : #MAFileDescription }
+
+{ #category : #'*Magritte-Pharo7-Model' }
+MAFileDescription >> defaultDirectory [
```

Comment:
I'm not sure that this is right to move this into the platform-specific package. IIUC this is part of the generic model and e.g. `MAFileSystem` would be the loaded elsewhere because it's platform-specific

Reply:
I did not now where to put it. I believe I move it to another package to satisfy "clean" package dependencies. And I do not remember to see any users (again, if I remember it properly... have not checked recently).